src/utils/helpers.ts

```diff
@@ -54,6 +58,28 @@ const getPlatformInfo = (platformName: string) => {
   else return { shortName: '', icon: {} };
 };
 
+const getDocsURL = (suite: string, framework_id: Framework['id']) => {
+  const framework = frameworkMap[framework_id];
+  const supportedFramework =
+    supportedPerfdocsFrameworks[framework as SupportedPerfdocsFramework];
+  const urlReadySuite = suite.replace(/:|\s|\.|_/g, '-').toLowerCase();
+
+  let docsURL = '';
+  let isLinkSupported = true;
+
+  const isDevToolsFramework = framework_id === 12;
+
+  if (isDevToolsFramework && supportedFramework) {
+    docsURL = `${baseDocsURL}/devtools/tests/${supportedFramework}.html#${urlReadySuite}`;
```

Comment:
There are some tests that are currently not in the documentation and you need to exclude them. 
export const [nonDocumentedTestsDevTools](https://github.com/mozilla/treeherder/blob/master/ui/perfherder/perf-helpers/perfdocs.js#L58) = [
  'reload-inspector:content-process',
  'reload-inspector:parent-process',
  'reload-debugger:content-process',
  'reload-debugger:parent-process',
  'reload-no-devtools:content-process',
  'reload-no-devtools:parent-process',
  'reload-netmonitor:content-process',
  'reload-netmonitor:parent-process',
  'reload-webconsole:parent-process',
  'reload-webconsole:content-process',
];

Reply:
Same for these [tests](https://github.com/mozilla/treeherder/blob/master/ui/perfherder/perf-helpers/perfdocs.js#L48C1-L55C3)